code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -602,27 +608,70 @@ Code:
 				. += "Pressure Inlet: [round(MIXTURE_PRESSURE(circ2?.air1), 0.1)] kPa  Outlet: [round(MIXTURE_PRESSURE(circ2?.air2), 0.1)] kPa<BR>"
 				. += "<BR>"
 
+		// SINGULO
 		if(length(by_type[/obj/machinery/power/collector_control]))
 			var/controler_index = 1
 			var/collector_index = 1
 			for_by_tcl(C, /obj/machinery/power/collector_control)
 				collector_index = 1
 				if(C?.active && C.z == 1)
 					engine_found = TRUE
-					. += "<BR><h4>Radiation Collector [controler_index++] Status</h4>"
+					. += "<h4>Radiation Collector [controler_index++] Status</h4>"
 					. += "Output: [engineering_notation(C.lastpower)]W<BR>"
 					if(C.CA1?.active) . += "Collector [collector_index++]: Tank Pressure: [C.P1 ? round(MIXTURE_PRESSURE(C.P1.air_contents), 0.1) : "ERR"] kPa<BR>"
 					if(C.CA2?.active) . += "Collector [collector_index++]: Tank Pressure: [C.P2 ? round(MIXTURE_PRESSURE(C.P2.air_contents), 0.1) : "ERR"] kPa<BR>"
 					if(C.CA3?.active) . += "Collector [collector_index++]: Tank Pressure: [C.P3 ? round(MIXTURE_PRESSURE(C.P3.air_contents), 0.1) : "ERR"] kPa<BR>"
 					if(C.CA4?.active) . += "Collector [collector_index++]: Tank Pressure: [C.P4 ? round(MIXTURE_PRESSURE(C.P4.air_contents), 0.1) : "ERR"] kPa<BR>"
 					. += "<BR>"
 
+		// NUKE
+		if(nuke_reactor)
+			engine_found = TRUE
+			/*
+			var/rodlevel = null
+			for(var/x=1 to length(nuke_reactor.component_grid))
+				if (!isnull(rodlevel))
+					return
+				for(var/y=1 to length(nuke_reactor.component_grid[1]))
+					if(nuke_reactor.component_grid[x][y])
+						if(istype(nuke_reactor.component_grid[x][y],/obj/item/reactor_component/control_rod))
+							var/obj/item/reactor_component/control_rod/CR = nuke_reactor.component_grid[x][y]
+							rodlevel = CR.configured_insertion_level
+							return
+			for some unfathomable reason this broke the whole damn thing, someone else can give it a crack if they like. */
```

Comment:
either fix this or remove it, big blocks of commented code shouldn't go in a PR

Reply:
good point, i'll shorten the comment